### PR TITLE
Fixed truncated dropdown menu

### LINKED ISSUE #1037: Truncated Dropdown Menu Items in the Sub-Articles Section
Summary: In the "Sub-Articles" section of the "Contributing" page, when the dropdown menu is expanded, some section names are cut off at the right edge of the page. This requires users to horizontally scroll the page to read them completely, which can lead to a poor user experience.

URL: https://www.pythondiscord.com/pages/guides/pydis-guides/contributing/

Environment: windows 10, display resolution: 1920х1080, browsers: Chrome, Firefox

Step to reproduse: 
1.  Navigate to the "Contributing" page.
2. Locate the "Sub-Articles" section at the top right of the page.
3. Click on the "Sub-Articles" dropdown menu to expand it.
4. Observe the section names at the right edge of the expanded dropdown menu.

Expected Result: All section names in the dropdown menu should be fully visible without requiring horizontal scrolling.

Actual Result: Some section names in the dropdown menu are cut off at the right edge of the page, requiring users to horizontally scroll to read them completely.

Attachment:

Contributing2

https://github.com/python-discord/site/assets/51853424/6b14ed2a-279b-4a53-acac-d12a52cb4d1b

### PR DESCRIPTION
**Pull request by sarzz2**
fixes #1037 

## Summary

In the "Sub-Articles" section of the "Contributing" page, when the dropdown menu is expanded, some section names are cut off at the right edge of the page making user scroll horizontally. 

## Description of changes

removed nowrap property and adjusted CSS of the subarticle to make it fit on the page

### I confirm I have:

- [X] Joined the [Python Discord community](discord.gg/python)
- [X] Read the [Code of Conduct](https://www.pydis.com/pages/code-of-conduct) and agree to it
- [X] I have discussed implementing this feature on the relevant service (Discord, GitHub, etc.)

### CODE REVIEW

**Review by hedyhli — COMMENTED** [review]
Thank you! This was bothering me for a long time.

One problem though, this doesn't fix it on mobile:
image

Maybe it's out of scope, I could fix this case in another PR.

However, here's my idea:

We can add two lines to the max width media query (in the same file), so the dropdown menu can then be fixed on the right edge:

image

On line 12, something like this:
```css
.dropdown-menu {
  left: unset;
  right: 0;
}
```

What do you think?

**Comment by sarzz2** [comment]
works fine @hedyhli committed the changes.

File: pydis_site/static/css/content/page.css [hunk 1]
```diff
@@ -96,3 +101,12 @@ ul.menu-list.toc {
 li img {
     margin-top: 0.5em;
 }
+
+.dropdown-menu{
+    min-width: 15rem;
+}
```
Comment (wookie184) [inline pydis_site/static/css/content/page.css]:
```suggestion
.dropdown-menu {
    min-width: 15rem;
}
```

**Review by wookie184 — APPROVED** [review]
All seems good to me, thanks

**Review by hedyhli — APPROVED** [review]
Works beautifully 👍

**Pull request merged**